app/controllers/opening_jobs_controller.rb

```diff
@@ -39,7 +41,7 @@ def interested_people
   end
 
   def jobs_list
-    @jobs = OpeningJob.all
+    @jobs = OpeningJob.all.where(open: true)
```

Comment:
```suggestion
    @jobs = OpeningJobl.where(open: true)
```
also create one scope for open jobs